electron/utils.ts

```diff
@@ -148,12 +148,32 @@ export const getUserInfo = (): UserInfo => {
   return { account_id: '', displayName: null }
 }
 
-const updateGame = (game: string) => {
+const updateGame = async (game: string) => {
   const logPath = `${heroicGamesConfigPath}${game}.log`
   const command = `${legendaryBin} update ${game} -y &> ${logPath}`
-  return execAsync(command, { shell: '/bin/bash' })
-    .then(console.log)
-    .catch(console.log)
+
+  try {
+    await execAsync(command, { shell: '/bin/bash' })
+  } catch (error) {
+    const noSpaceMsg = 'Not enough available disk space'
+    const genericMessage = 'installation canceled or had some error'
+    await execAsync(`tail ${logPath} | grep 'disk space'`)
+      .then(({ stdout }) => {
+        if (stdout.includes(noSpaceMsg)) {
+          console.log(noSpaceMsg)
+          return showErrorBox(
+            i18next.t('box.error.diskspace.title', 'No Space'),
+            i18next.t(
+              'box.error.diskspace.message',
+              'Not enough available disk space'
+            )
+          )
+        }
+        console.log(genericMessage)
+        return genericMessage
+      })
+      .catch(() => console.log(genericMessage))
+  }
```

Comment:
This block of code is VERY similar to the install code ([electron/main.ts:365-387](https://github.com/flavioislima/HeroicGamesLauncher/pull/236/files#diff-13ee644059574b054a4e5ec081666a03579b28da03a50331215908f8d0ef7cfdR365-R387)). Maybe its an idea to unify them for ease of maintainability? Also right now the install and update are in 2 different files which makes it also harder to maintain.

Reply:
Yeah, I'll create one single function to avoid repetition